modules/assim.batch/R/pda.emulator.R

```diff
@@ -218,20 +221,23 @@ pda.emulator <- function(settings, params.id=NULL, param.names=NULL, prior.id=NU
     }else{
       load(settings$assim.batch$emulator.path) # load previously built emulator to run a longer mcmc
       load(settings$assim.batch$llik.path)
-      load(settings$assim.batch$jvar.path)
-      load(settings$assim.batch$mcmc.path)
+      load(settings$assim.batch$resume.path)
+      #load(settings$assim.batch$mcmc.path)
       
-      init.list <- list()
+      prior.all <- do.call("rbind", prior.list)
+      prior.ind.all <- which(unlist(pname) %in% unlist(settings$assim.batch$param.names))
+      prior.fn.all <- pda.define.prior.fn(prior.all)
       
+
       for(c in 1:settings$assim.batch$chain){
-        init.x <- mcmc.list[[c]][nrow(mcmc.list[[c]]),]
-        
-        prior.all <- do.call("rbind", prior.list)
-        prior.ind.all <- which(unlist(pname) %in% unlist(settings$assim.batch$param.names))
-        prior.fn.all <- pda.define.prior.fn(prior.all)
-        
-        init.list[[c]] <-  as.list(sapply(seq_along(prior.ind.all), 
-                                          function(x) eval(prior.fn.all$pprior[[prior.ind.all[x]]], list(q=init.x[x]))))
+        init.list[[c]] <- resume.list[[c]]$prev.samp[nrow(resume.list[[c]]$prev.samp),]
+        jmp.list[[c]] <- resume.list[[c]]$jump
+        # init.x <- mcmc.list[[c]][nrow(mcmc.list[[c]]),]
```

Comment:
yes, it was left from the previous implementation, now I erased the unused commented-out code bits